libafl_libfuzzer_runtime/build.ps1

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env pwsh

```

Comment:
Maybe call that guy from a justfile?

Reply:
Good idea, I'll add a justfile for libafl_libfuzzer and libafl_libfuzzer_runtime.